packages/web/src/services/audio-player/NativeMobileAudio.ts

```diff
@@ -10,13 +10,15 @@ import { generateM3U8Variants } from 'utils/hlsUtil'
 
 const PUBLIC_IPFS_GATEWAY = 'http://cloudflare-ipfs.com/ipfs/'
 
-class NativeMobileAudio {
+export class NativeMobileAudio {
+  audio: any
   m3u8: string
   position: number
   duration: number
   // Whether or not the user has made a seek action.
   // We use this to make sure we don't read stale position values from the native layer.
   seekOverride: number | null
+  audioCtx = null
```

Comment:
this doesn't carry any meaning in native land, but i guess this file goes away entirely anyway right?

Reply:
yeah just to get types to be happy, and yes goes away in native-reloaded